### DISCUSSION TITLE
Question about timedomain

### DISCUSSION THREAD

**Discussion by harryxhr** [lululxvi/deepxde#809] [Q&A] [answered]
Hi, Dr Lu lu
i am a student trying to use DeepXDE to solve ODEs. Really thanks for your great work. 
It seems like the PINN only valid for small interval. i am working on van der pol equation. if i use the Timedomain as [0,5], the results is very accurate. But when i increase the domain to [0,30] the prediction results converge to zero very quickly which is not correct.
I am not sure if is common problem for PINN as it will perform bad when using large interval? Is there someways to fix this?
Best regards

**Comment by lululxvi** [comment, marked as answer]
There are some ways to improve it, but it is still an open problem for PINN.

**Reply by haoranxux** [reply]
Thanks so much for your reply!
